src/e2e/azureDevOps.test.ts

```diff
@@ -0,0 +1,119 @@
+import AzureDevOpsCommand from '../cli/commands/azure-devops';
+import { captureOutput, OutputMock, suppressOutput } from './helpers/cli';
+import GitHelper from './helpers/git';
+import AzureDevOpsHelper, {
+  loadAzureDevOpsTestEnv,
+} from './helpers/azureDevOps';
+
+describe('AzureDevOps', () => {
+  jest.setTimeout(30_000);
+
+  let token: string;
+  let repoUrl: string;
+  let az: AzureDevOpsHelper;
+  let git: GitHelper;
+  let branch: string;
+  let prNumber: number;
+  let out: OutputMock;
+  let args: string[];
+
+  beforeAll(async () => {
+    const env = loadAzureDevOpsTestEnv();
+    repoUrl = env.repoUrl;
+    token = env.token;
+
+    az = new AzureDevOpsHelper(repoUrl, token);
+    git = new GitHelper(repoUrl, '', token);
+
+    await az.createRepoIfNotExists();
+    await git.cloneTemplateRepo();
+    [branch] = await git.createBranch();
+    prNumber = await az.createPr(branch);
+
+    suppressOutput();
+  });
+
+  afterAll(async () => {
+    await az.closePr(prNumber);
+    await git.deleteBranch(branch);
+    await git.cleanupRepo();
+  });
+
+  beforeEach(() => {
+    out = new OutputMock();
+    captureOutput(out);
+  });
+
+  afterEach(jest.clearAllMocks);
+  afterAll(jest.restoreAllMocks);
+
+  describe('PR', () => {
+    beforeAll(async () => {
+      // Add non-matching existing comment
+      args = [repoUrl, 'pr', `${prNumber}`, `--azure-devops-token=${token}`];
+
+      await AzureDevOpsCommand.run([
+        'new',
+        ...args,
+        '--body=existing',
+        '--tag=existing',
+      ]);
+    });
+
+    test('test', () => {
+      expect(true).toBeTruthy();
+    });
```

Comment:
Just of curiosity why do we need this test?

Reply:
Didn't mean to leave that in, thanks!